terraform/stacks/umccr_data_portal/pipeline/oncoanalyser.tf

```diff
@@ -33,6 +33,10 @@ resource "aws_cloudwatch_event_rule" "oncoanalyser" {
       "jobQueue" : [
         { "prefix" : "arn:aws:batch:${local.region}:${local.account_id}:job-queue/nextflow-pipeline" }
       ],
+      "parameters": {
+        "orcabus": [ { "exists": false } ],
```

Comment:
That would not conver cases where orcabus = false I guess, but since we don't have any of those any time soon this should work nicely!

Reply:
Yup